README.md

```diff
@@ -1080,9 +1081,6 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2019. [Pro
 - <a name="Miao19fnbot"></a>Miao, S., Chen, G., Ning, X., Zi, Y., Ren, K., Bing, Z., Knoll, A.,  
 *[Neuromorphic Vision Datasets for Pedestrian Detection, Action Recognition, and Fall Detection](https://doi.org/10.3389/fnbot.2019.00038),*  
 Front. Neurorobot. (2019). [Dataset](https://github.com/MSZTY/PAFBenchmark) 
```

Comment:
It is good to have the dataset here, but not with a duplicate link. Just a link to the entry in pattern recognition section. As you wish.